ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -83,6 +102,7 @@ export default Route.extend({
           pageFilter: params.pageFilter,
         })
         .then(model => {
+          console.log({ model });
```

Comment:
just a heads up about the console log.  I know this is still a draft, but wanted to point it out.